crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -4726,14 +4751,12 @@ mod tests {
         );
     }
 
-    #[track_caller]
-    fn assert_scope_ty(
-        db: &TestDb,
+    fn get_symbol<'db>(
```

Comment:
This function also has a bunch of assertions and other places where it could panic; maybe it's worth adding `#[track_caller]` to this as well?

```suggestion
    #[track_caller]
    fn get_symbol<'db>(
```